File: Test/Test/iOS/Objective-C/Practice/ContactFetching/ContactFetching/ViewController.m
//
//  ViewController.m
//  ContactFetching
//
//  Created by Balaji Goud  on 4/20/17.
//  Copyright © 2017 RBT. All rights reserved.
//

#import "ViewController.h"
#import "SystemContacts.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [SystemContacts getAllContactsWithCompletionBlock:^(NSMutableArray *arrayContacts) {
        NSLog(@"%@",arrayContacts);
    }];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Test/Test/iOS/Swift/Practice/ContinueLabel/ContinueLabel/ViewController.swift
//
//  ViewController.swift
//  ContinueLabel
//
//  Created by Balaji Goud  on 4/5/17.
//  Copyright © 2017 RBT. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var lblTwo: UILabel!
    @IBOutlet weak var lblOne: UILabel!
    override func viewDidLoad() {
        super.viewDidLoad()
        let String = "You use a defer statement to execute a set of statements just before code execution leaves the current block of code. This statement lets you do any necessary cleanup that should be performed regardless of how execution leaves the current block of code—whether it leaves because an error was thrown or because of a statement such as return or break. For example, you can use a defer statement to ensure that file descriptors are closed and manually allocated memory is freed. A defer statement defers execution until the current scope is exited. This statement consists of the defer keyword and the statements to be executed later. The deferred statements may not contain any code that would transfer control out of the statements, such as a break or a return statement, or by throwing an error. Deferred actions are executed in reverse order of how they are specified—that is, the code in the first defer statement executes after code in the second, and so on."
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

